server/events/models/models.go

```diff
@@ -423,7 +423,7 @@ func (p ProjectCommandContext) ProjectCloneDir() string {
 // SetScope sets the scope of the stats object field. Note: we deliberately set this on the value
 // instead of a pointer since we want scopes to mirror our function stack
 func (p ProjectCommandContext) SetScope(scope string) {
-	p.Scope = p.Scope.Scope(scope)
+	p.Scope = p.Scope.Scope(scope) //nolint
```

Comment:
Why'd you add this tag?

Reply:
I'm getting this warning without the tag `ineffective assignment to field ProjectCommandContext.Scope (SA4005)go-staticcheck`. I looked up the error code SA4005 and this is what I found `SA4005 – Field assignment that will never be observed. Did you mean to use a pointer receiver`. I'm not sure what it means.

Reply:
Looks like it's passing all the tests without the //nolint tag as well. So, I have removed it.

Reply:
oh i think you found a bug of mine 😅 . I can address in another PR though.

Reply:
Sure thing!